### ISSUE TITLE
Hope and Gear not getting removed from PC after a Push

### ISSUE THREAD

**Issue by jwmuk** [erizocosmico/foundry-coriolistgd#2]
I've created new characters with v0.3.2 but when I push a roll and the pushed roll has a 1 for Gear or Base Attributes, it doesn't automatically remove these from hope or my gear

**Comment by erizocosmico** [comment]
Yup, that automation is not implemented yet, but it's on the bucket list. It'll be added soon

**Comment by erizocosmico** [comment]
This is implemented now in v0.3.3
